pytest-profiling/setup.cfg

```diff
@@ -9,4 +9,4 @@ norecursedirs =
 	tests/integration/profile/tests
 
 [bdist_wheel]
-universal = 1
+universal = 0
```

Comment:
@jonbannister why did you change it? Did the library start shipping C-extensions instead of pure-python?